target/linux/ipq807x/base-files/etc/hotplug.d/firmware/11-ath11k-caldata

```diff
@@ -21,23 +21,26 @@ case "$FIRMWARE" in
 	zyxel,nbg7815)
 		caldata_extract "0:art" 0x1000 0x20000
 		;;
+	yuncore,ax880)
+		caldata_extract "0:ART" 0x1000 0x20000
+		;;
 	prpl,haze)
-                caldata_extract_mmc "0:ART" 0x1000 0x20000
+		caldata_extract_mmc "0:ART" 0x1000 0x20000
```

Comment:
I am sure this is unintended, remove the change.

Reply:
I don't get it.

Case blocks content is intended now:
https://github.com/openwrt/openwrt/blob/ae500e62e2938e112ae1fc6aa7389e8c7b784b13/target/linux/qualcommax/ipq807x/base-files/etc/hotplug.d/firmware/11-ath11k-caldata#L33

I have replaced all spaces with tabs (some of them are present in this file at the moment https://github.com/openwrt/openwrt/blob/ae500e62e2938e112ae1fc6aa7389e8c7b784b13/target/linux/qualcommax/ipq807x/base-files/etc/hotplug.d/firmware/11-ath11k-caldata#L37C15-L37C15).